cli/README.md

```diff
@@ -46,17 +46,15 @@ the RACK-Box README.
 ```shell
 python3 -m venv venv
 source venv/bin/activate
-pip install --force -r requirements.txt
-python3 setup.py install
+pip install --force-reinstall .
```

Comment:
Don't you need `pip3`?

Reply:
The environments I've been working in *only* have pip version 3, and pip points to that. I'll change the documentation to be explicit about being version 3 for platforms where version 2 still exists